### PR TITLE
add kubernetes deploy example

### PR DESCRIPTION
**Pull request by TheKangaroo**
I deployed the heartbeat proxy to kubernetes yesterday and wanted to share the examples with you.

### CODE REVIEW

File: examples/k8sdeploy.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+spec:
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: heartbeat-proxy
+    spec:
+      containers:
+        - name: heartbeat-proxy
+          image: registry.gitlab.dm-drogeriemarkt.com/docker/opsgenie-heartbeat-proxy
```
Comment (jayme-github) [inline examples/k8sdeploy.yaml]: Could you please exchange the `image` for the autobuild one from https://hub.docker.com/r/traumfewo/opsgenie-heartbeat-proxy/

File: examples/k8sdeploy.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    app: heartbeat-proxy
+  name: heartbeat-proxy
+  namespace: monitoring
```
Comment (jayme-github) [inline examples/k8sdeploy.yaml]: Please remove the `namespace:` to make this a little more generic.

**Review by jayme-github — CHANGES_REQUESTED** [review]
Hi,
thank you very much for your contribution!

I will forward this to @NilsGriebner for further review and merge. Do you mind adding a small note to the README regarding your example?

**Comment by NilsGriebner** [comment]
@TheKangaroo thanks for your contribution :) I agree with @jayme-github points. The changes would be great. We use resource limits in our deployment, maybe we should add it here too. What do you think?

File: examples/k8sdeploy.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    spec:
+      containers:
+        - name: heartbeat-proxy
+          image: registry.gitlab.dm-drogeriemarkt.com/docker/opsgenie-heartbeat-proxy
+          env:
+          - name: OPSGENIE_API_KEY
+            value: "youropsgenieapikey"
+          - name: HEARTBEAT_NAME
+            value: "yourheartbeatname"
+#          - name: DEBUG
```
Comment (NilsGriebner) [inline examples/k8sdeploy.yaml]: I think this could be removed, because it's for debugging only.

**Review by NilsGriebner — CHANGES_REQUESTED** [review]

**Comment by TheKangaroo** [comment]
Hey @NilsGriebner, @jayme-github , sorry for the delay. I fixed the image path and removed the namespace.
I also added the limits. We use them too in our setup, but I'm not 100% sure if we really should add them to a simple example. I'm fine with both ways, you guys decide ;-)

I added some lines about kubernetes to the readme too.
What do you think?

**Comment by NilsGriebner** [comment]
Hi @TheKangaroo looks good to me now :) I will give it a try in our playground and merge if I don't spot any issues.

Thanks aggain, great work :)

**Review by NilsGriebner — APPROVED** [review]

**Pull request merged**